roles/n8n/tasks/main.yaml

```diff
@@ -0,0 +1,22 @@
+- name: Create n8n user with
+  when: n8n_db_user is defined and n8n_db_password is defined
+  become: true
+  become_user: postgres
+  postgresql_user:
+    state: present
+    db: "{{ db }}"
+    user: "{{ n8n_db_user }}"
+    password: "{{ n8n_db_password }}"
+    role_attr_flags: NOLOGIN
```

Comment:
Why? Doesn't the user need to log in? Or is that only temporary until you did the IP restriction?

Reply:
Yep, I wanted to be sure no one can login through this one. But it needs to be deleted. 
I guess my whole PR should be deleted as well since yours is better.